<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>861</x>
     <y>1246</y>
     <width>296</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSwitchValue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开关变量值：0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSwitch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开开关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddControllers">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>100</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加控制器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnM1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_M1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>M点1的值: 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnM2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_M2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>M点2的值: 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_M3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>M点3的值: 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnM3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnectAndSend">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接到控制器并发送数据</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_controllers">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_M3_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已添加的控制器信息</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
